malwarecage/web/src/components/ShowSample.js

```diff
@@ -10,15 +10,22 @@ import { Extendable } from "@malwarefront/extensions";
 import { Identicon, DataTable, DateString, Hash, View } from "@malwarefront/ui";
 import ShowObjectPresenter, {joinActions} from './ShowObjectPresenter';
 import ShowSamplePreview from "./ShowSamplePreview";
-
+import ActionCopyToClipboard from "./ActionCopyToClipboard";
 
 function SampleDetails(props) {
     return (
-        <DataTable>
-            <Extendable ident="showSampleDetails" object={props}>
+        <DataTable className="flicker-element">
+            <Extendable ident="showSampleDetails" object={props} className="flicker-element">
                 <tr>
                     <th>Filename</th>
-                    <td id="file_name"><Link to={makeSearchLink("name", props.file_name, false, '')}>{props.file_name}</Link>
+                    <td id="file_name">
+                        <Link to={makeSearchLink("name", props.file_name, false, '')}>
+                            {props.file_name}
+                        </Link>
+                        <small className="ml-2 mr-1 mt-1">
```

Comment:
- `small` isn't to small? Actually, regular span looks quite ok.
- Why we have `mr-1 mt-1` here? They're missing in other fields.